### DISCUSSION TITLE
LTSP Client Desktop doesn't load

### DISCUSSION THREAD

**Discussion by vinalcala** [ltsp/ltsp#844] [Q&A]
Hi,

I installed LTSP on Ubuntu Mate 20.04, i setup the LTSP server and also the client, it was running well and smooth on the server and client side. 1 week later, as i booted the client and logged in its user, the Desktop doesn't load its content, it only shows the wallpaper and the top panel and the loading cursor.  it wont click anything. But sometimes it was okay when i logged the user in, but still later on when i logged another user back in again, it happens something like that.  Hoping for your opinion on this issue. thanks

**Comment by alkisg** [comment]
Since this is a new installation, I'd recommend installing 22.04 from scratch instead of trying to troubleshoot 20.04.
It could be a hardware networking issue though, completely unrelated to software...
